buy-resell/products/3rd-party/datev-meinfiskal.md

```diff
@@ -79,7 +100,7 @@ Please note that the DATEV MeinFiskal account is created automatically during th
 
   </details>
 
-4. The DATEV MeinFiskal dialog box for changing the password is displayed. The PosDealer sets a new password and clicks "Save".
+* 4. The _DATEV MeinFiskal_ dialog box for changing the password is displayed. The PosDealer sets a new password and clicks **Save**.

```

Comment:
The numbering is not the same as in previous cases (1,2,3). And in this case, we are showing IV.